pkg/backend/httpstate/client/client.go

```diff
@@ -305,6 +305,52 @@ type serviceTokenInfo struct {
 	Team         string `json:"team,omitempty"`
 }
 
+//nolint:gosec // These are token type URNs, not credentials
+const (
+	pulumiAccessTokenTypeOrganization = "urn:pulumi:token-type:access_token:organization"
+	pulumiAccessTokenTypeTeam         = "urn:pulumi:token-type:access_token:team"
+	pulumiAccessTokenTypePersonal     = "urn:pulumi:token-type:access_token:personal"
+)
+
+func (pc *Client) ExchangeOidcToken(
+	oidcToken string, org string, scope string, expiration int64,
```

Comment:
Probably gonna make this comment in a few places but can we prefer using time.Time/time.Duration types for passing timing information around and only converting to integer seconds at wire protocol boundaries.